.gitignore

```diff
@@ -31,3 +31,8 @@ yarn-error.log*
 # generated
 .next
 .vscode
+
+# misc
+yarn.lock
```

Comment:
Will do that, I encountered a merge conflict and I was not sure if it was forcing specific changes from my local environment because It took hours of set up to make supabase cli work on my local upon the project and to the team. `In simple term I feel my environment and corrupted and insane and I did not want to infect the rest of you by pushing those changes to the repo lol`